source/plugin/commands/arguments.rst

```diff
@@ -10,7 +10,11 @@ Argument Parsing
     org.spongepowered.api.command.args.CommandContext
     org.spongepowered.api.command.args.CommandElement
     org.spongepowered.api.command.args.GenericArguments
+    org.spongepowered.api.command.args.SelectorCommandElement
```

Comment:
To you use this import somewhere?

Reply:
My original version did, just left this in.